### PR TITLE
Make analytics opt-out #added

### PR DESCRIPTION
**Pull request by fulldecent**
## Changes:
- Makes analytics collection opt-out

## Closes following issues:
- Closes: #1557

## Tested:
- Processors:
  - [ ] Intel
  - [x] Apple Silicon
- macOS Versions:
  - [ ] 10.12.x (Sierra)
  - [ ] 10.13.x (High Sierra)
  - [ ] 10.14.x (Mojave)
  - [ ] 10.15.x (Catalina)
  - [ ] 11.x (Big Sur)
  - [x] 12.x (Monterey)
- Localizations:
  - [x] English
  - [ ] Spanish
  - [ ] Other (please specify)
- Xcode Version: 14.0
  
## Screenshots:

Screen Shot 2022-09-22 at 22 42 40

### CODE REVIEW

File: Resources/Plists/PreferenceDefaults.plist [hunk 1]
```diff
@@ -101,6 +101,8 @@
 	<integer>20</integer>
 	<key>EditInSheetForMultiLineText</key>
 	<true/>
+	<key>SaveApplicationUsageAnalytics</key>
+	<true/>
```
Comment (Kaspik) [inline Resources/Plists/PreferenceDefaults.plist:105]: Makong sure - this means ON by default, right?

File: Source/Interfaces/Preferences.xib [hunk 2]
```diff
@@ -755,10 +755,43 @@ Gw
...
+                        <color key="textColor" name="controlTextColor" catalog="System" colorSpace="catalog"/>
+                        <color key="backgroundColor" name="controlColor" catalog="System" colorSpace="catalog"/>
+                    </textFieldCell>
+                </textField>
+                <button translatesAutoresizingMaskIntoConstraints="NO" id="2aR-JM-Lbr">
+                    <rect key="frame" x="178" y="16" width="342" height="24"/>
+                    <constraints>
+                        <constraint firstAttribute="height" constant="22" id="f9S-3k-CY3"/>
+                    </constraints>
+                    <buttonCell key="cell" type="check" title="Send application usage analytics" bezelStyle="regularSquare" imagePosition="left" alignment="left" inset="2" id="mZ1-oa-LuT">
```
Comment (Jason-Morcos) [inline Source/Interfaces/Preferences.xib:775]: Slight tweak on language:
"Share anonymized usage analytics with developers" or something like that

Comment (fulldecent) [reply Source/Interfaces/Preferences.xib]: Updated at ab631f83889c973d605395481d64e3c23d6ba853

**Review by Kaspik — APPROVED** [review]
My Q is still otherwise LGTM.

File: Resources/Plists/PreferenceDefaults.plist (hunk 1 above)
Comment (fulldecent) [reply Resources/Plists/PreferenceDefaults.plist:105]: That is correct

**Comment by Lenitr** [comment]
Just saw this feature in the changelog, but I weirdly don't see this checkbox (even though I'm on version 4.0.0)

Screenshot 2022-10-20 at 21 54 44

I have installed Sequel Ace through the App store, could that have something to do with it?

**Comment by Kaspik** [comment]
It shouldn't have anything to do with AppStore.

**Comment by Jason-Morcos** [comment]
Interesting. I am seeing the same thing as well. The build number is wrong too on the App Store release - should be 20037. It's like somehow the previous release was published instead of the very latest?

**Comment by Jason-Morcos** [comment]
I probably made a mistake in the publishing process 🤦

**Comment by Lenitr** [comment]
Good to hear you've been able to reproduce the issue though, thanks for your time 🙂

**Comment by Jason-Morcos** [comment]
4.0.1 submitted to App Store which should actually include _all_ the features of 4.0.0. Thanks for letting us know about this, @Lenitr!

https://github.com/Sequel-Ace/Sequel-Ace/releases/tag/production%2F4.0.1-20038

**Comment by Lenitr** [comment]
Thanks a lot, I'll try it whenever the App Store detects the new version (I guess it's still going through Apple's verifications etc. before it's available)

**Comment by Lenitr** [comment]
Just got the update through the Apple store, can confirm that this version shows the new feature, thanks again 👍🏼

**Pull request merged**